### PR TITLE
rtracklayer changes for BiocIO

### PR DESCRIPTION
**Pull request by dvantwisk**
Here is a pull request for the changes that I mentioned in issue https://github.com/lawremi/rtracklayer/issues/25. Note that BiocIO must be installed for it to work properly https://github.com/Bioconductor/BiocIO.

### CODE REVIEW

File: R/igv.R [hunk 1]
```diff
@@ -13,7 +13,7 @@ handleCategory <- function(x) {
     categories <- lapply(getNodeSet(x, "Category"), handleCategory)
     children <- do.call(rbind, categories)
     children$parenet <- name
-    resources <- rtracklayer:::RTLFileList(lapply(paths, FileForFormat))
+    resources <- rtracklayer:::BiocFileList(lapply(paths, FileForFormat))
```
Comment (lawremi) [inline R/igv.R:16]: We should probably drop the `rtracklayer:::`. Probably wasn't supposed to be there in the first place.

**Comment by dvantwisk** [comment]
I've resolved the small issue that you've mentioned. I think we can hold off merging this change to devel until BiocIO gets placed on devel as well. Thank you for looking over this.

**Comment by lawremi** [comment]
Ok, thanks, please let me know when this is ready to be merged.

**Comment by Kayla-Morrell** [comment]
Hello @lawremi - `BiocIO` has now been accepted and added to the manifest.

```{r}
> BiocManager::install("BiocIO")
Bioconductor version 3.12 (BiocManager 1.30.10), R 4.0.0 alpha (2020-04-07
  r78171)
Installing package(s) 'BiocIO'
trying URL 'https://bioconductor.org/packages/3.12/bioc/src/contrib/BiocIO_0.99.11.tar.gz'
Content type 'application/x-gzip' length 230957 bytes (225 KB)
==================================================
downloaded 225 KB

* installing *source* package ‘BiocIO’ ...
** using staged installation
** R
** inst
** byte-compile and prepare package for lazy loading
** help
*** installing help indices
** building package indices
** installing vignettes
** testing if installed package can be loaded from temporary location
** testing if installed package can be loaded from final location
** testing if installed package keeps a record of temporary installation path
* DONE (BiocIO)

The downloaded source packages are in
	‘/private/var/folders/sk/hy088prx12l_cqspv3lbpl9s6_3r11/T/RtmpynSM26/downloaded_packages’
> library(BiocIO)
>
```

This PR should be good to get merged now. Let me know if there are any other changes that need to be made.

Best,
Kayla

**Comment by lawremi** [comment]
Thanks a lot for taking care of the update for me. This refactoring is definitely a step in the right direction.

**Comment by Kayla-Morrell** [comment]
Hello @lawremi - Now that the release is over we were hoping it might be a good time to get this PR merged over. I've already begun contacting potentially affected reverse dependency maintainers in hopes of aiding in the switch over to BiocIO. Please let me know if there is anything further you need from us. Thanks!

**Pull request merged**